haiku-apps/beae/beae-1.2.recipe

```diff
@@ -5,9 +5,10 @@ HOMEPAGE="https://github.com/HaikuArchives/BeAE"
 COPYRIGHT="2003 Xentronix"
 LICENSE="BSD (3-clause)"
 REVISION="4"
-SOURCE_URI="https://github.com/HaikuArchives/BeAE/archive/v1.2.tar.gz"
+SOURCE_URI="https://github.com/HaikuArchives/BeAE/archive/v$portVersion.tar.gz"
 CHECKSUM_SHA256="d69a16821f86eda6e94b8297928f9def75f5c750e1e7d6b946d0b195fdc88943"
-SOURCE_DIR="BeAE-1.2"
+SOURCE_DIR="BeAE-$portVersion"
+SOURCE_FILENAME="BeAE-$portVersion.tar.gz"
 
 ARCHITECTURES="x86_gcc2 !x86 ?x86_64"
```

Comment:
Complicated: teams should contain team_id instances as in "new team_id". These should be deleted on BList destruction. ItemAt() return value can then be casted to "team_id*", then dereferenced.